src/networking.c

```diff
@@ -3372,6 +3372,30 @@ sds catClientInfoString(sds s, client *client, int hide_user_data) {
     return ret;
 }
 
+/* Concatenate a string representing the state of a client in a human
+ * readable format, into the sds string 's'.
+ *
+ * This is a simplified and shortened version of catClientInfoString,
+ * it only added some basic fields for tracking clients. */
+sds catClientInfoShortString(sds s, client *client, int hide_user_data) {
+    if (!server.crashed) waitForClientIO(client);
+    char conninfo[CONN_INFO_LEN];
+
+    sds ret = sdscatfmt(
+        s,
+        FMTARGS(
+            "id=%U", (unsigned long long)client->id,
+            " addr=%s", getClientPeerId(client),
+            " laddr=%s", getClientSockname(client),
+            " %s", connGetInfo(client->conn, conninfo, sizeof(conninfo)),
+            " name=%s", hide_user_data ? "*redacted*" : (client->name ? (char *)client->name->ptr : ""),
+            " cmd=%s", client->lastcmd ? client->lastcmd->fullname : "NULL",
```

Comment:
maybe the cmd is not required in this specific case as we already include that data in the log msg?

Reply:
Yes, if we add so many fields, we can even log the full client info?

We already have a short version as `"Error accepting a client connection: %s (addr=%s laddr=%s)"`. Is it not enough?

If we add another semi-short version, we have 3 variants of client info. Maybe it's too many variants?

Reply:
I want this short client info to be self-consistent, like it contain these fields but not the other stats fields. These fields are what i think are helpful to view, i can drop it (name / cmd / user) if you guys insist.

`"Error accepting a client connection: %s (addr=%s laddr=%s)"`. I somehow don't think it is a short client info, it is somehow like a connection info, but i guess it can be treated as client info as well.

Reply:
@enjoy-binbin I agree, but the cmd is not related to client identification right?

Reply:
right, i am removing it.